### PR TITLE
Refractor the empty role policy

### PR DESCRIPTION
**Pull request by athiruma**
## Type of change
Note: Fill **x** in []
- [ ] bug
- [x] enhancement
- [ ] documentation
- [ ] dependencies

## Description
Refractor the empty role to follow new approach.

### CODE REVIEW

**Review by ebattat — APPROVED** [review]
/LGTM

File: cloud_governance/policy/aws/empty_roles.py [hunk 1]
```diff
@@ -1,54 +1,54 @@
...
     def __init__(self):
         super().__init__()
 
-    def run(self):
-        """
-        This method returns all empty roles, delete if dry_run no
-        @return:
+    def run_policy_operations(self):
         """
-        return self.__delete_empty_roles()
```
Comment (ebattat) [inline cloud_governance/policy/aws/empty_roles.py:18]: return self.__delete_empty_roles() - is it ok that we remove it ?

Comment (athiruma) [reply cloud_governance/policy/aws/empty_roles.py:18]: yes, this is the old method

Comment (ebattat) [reply cloud_governance/policy/aws/empty_roles.py:18]: @athiruma , Pls fix conflicts

Comment (athiruma) [reply cloud_governance/policy/aws/empty_roles.py:18]: Done

**Review by ebattat — APPROVED** [review]

**Pull request merged**